class-jetpackbetaclicommand.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+if ( defined( 'WP_CLI' ) && WP_CLI ) {
+	/**
+	 * Control your local Jetpack Beta Tester plugin.
+	 */
+	class JetpackBetaCliCommand extends WP_CLI_Command {
+		/**
+		 * Activate a branch version
+		 *
+		 * ## OPTIONS
+		 *
+		 *
+		 * activate master: Get a version of the master branch built every 15 minutes
+		 *
+		 * activate stable: Get the latest stable version of Jetpack
+		 *
+		 * ## EXAMPLES
+		 *
+		 * wp jetpack-beta branch activate master
+		 * wp jetpack-beta branch activate stable
+		 *
+		 */
+		public function branch( $args ) {
+
+			$this->validation_checks($args);
```

Comment:
Should we check and so do something when errors that are returned here?

Reply:
actually [`WP_CLI::error`](https://make.wordpress.org/cli/handbook/internal-api/wp-cli-error/) return `null`, and exit the script right away. Your suggestion might be possible if `validation_checks` won't use `WP_CLI::error` but instead will return for example error string. But I don't think it worth it.